### PR TITLE
Refactor for the card theme

### PR DESCRIPTION
**Pull request by kkosiorowska**
This PR standardizes our styles for the `Card` component. What has been done:

- In many places we have set `padding={0}` for the `CardBody`. Let's set a single common padding for the Card component.
- Updated header styles for the cards.
- Removed the border for the `Card` component and the unneeded `elevated` variant.
- Removed unneeded props that are already defined in the card theme.

⚠️ Important notice ⚠️
Many things probably still need to be improved, but I believe this PR cleans our code a bit. Let's make the next fixes and integrate with styleguide in the next steps.

### CODE REVIEW

File: dapp/src/theme/Card.ts [hunk 1]
```diff
@@ -1,40 +1,29 @@
...
-  },
-  defaultProps: {
-    size: "lg",
-  },
-}
+const baseStyleContainer = defineStyle({
+  boxShadow: "none",
+  bg: "gold.200",
+  borderRadius: "xl",
+  padding: 5,
```
Comment (kpyszkowski) [inline dapp/src/theme/Card.ts:10]: Please use shorthand instead
```suggestion
  p: 5,
```

File: dapp/src/theme/Card.ts [hunk 2]
```diff
@@ -1,40 +1,29 @@
...
+})
+
+const baseStyleHeader = defineStyle({
+  padding: 0,
```
Comment (kpyszkowski) [inline dapp/src/theme/Card.ts:14]: Same as above

File: dapp/src/theme/Card.ts [hunk 3]
```diff
@@ -1,40 +1,29 @@
...
+})
+
+const baseStyleBody = defineStyle({
+  padding: 0,
```
Comment (kpyszkowski) [inline dapp/src/theme/Card.ts:18]: Same as above

**Review by kpyszkowski — COMMENTED** [review]
Left non blocking comments

File: dapp/src/theme/Card.ts (hunk 2 above)
Comment (kkosiorowska) [reply dapp/src/theme/Card.ts]: https://github.com/thesis/acre/pull/838/commits/dc231b83adad68ad3c5c2182854f0598fc36c11d

File: dapp/src/theme/Card.ts (hunk 1 above)
Comment (kkosiorowska) [reply dapp/src/theme/Card.ts]: https://github.com/thesis/acre/pull/838/commits/dc231b83adad68ad3c5c2182854f0598fc36c11d

**Review by kpyszkowski — APPROVED** [review]

**Pull request merged**